pkg/cmd/cli/secrets/satoken.go

```diff
@@ -0,0 +1,116 @@
+package secrets
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/client/unversioned"
+
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+)
+
+const (
+	ExtractSATokenRecommendedName = "extract-sa-token"
+
+	extractSATokenShort = `Extract the token from mountable secrets assigned to a ServiceAccount.`
+
+	extractSATokenLong = `
+Extract the token from mountable secrets assigned to a ServiceAccount.
+
+If the ServiceAccount has multiple tokens, the first token found will be extracted.
```

Comment:
Since manual editing of resources is supported, absolutely. Do we have a better way of determining which town to use if many exist? Why would many exist? I understand it can happen but I'm not sure how/why.